docs/manual/comparison_with_sql.rst

```diff
@@ -0,0 +1,662 @@
+.. py:currentmodule:: datatable
+
+Comparison with SQL
+====================
+
+This page provides some examples of how various `SQL <https://en.wikipedia.org/wiki/SQL>`__ operations can be performed in ``datatable``. The ``datatable`` library is still growing; as such, not all functions in ``SQL`` can be replicated yet. If there is a feature you would love to have in ``datatable``, please make a feature request on the `github issues <https://github.com/h2oai/datatable/issues>`__ page.
+
+Most of the examples will be based on the famous `iris <https://en.wikipedia.org/wiki/Iris_flower_data_set>`__ dataset. The assumption here is that the data is already loaded into a database; we'll simply show the relevant ``SQL`` queries side by side with ``datatable`` constructs.
+
+Let's import ``datatable`` and read in the data:
+
+.. code-block:: python
+
+    from datatable import dt, f, g, by, join, sort, update, fread
+
+    url = 'https://bit.ly/3kTNxM4'
```

Comment:
I wouldn't use `bit.ly` here, because by looking at the link it is impossible to guess where it points to (without clicking on it) and also, if `bit.ly` goes down temporarily or permanently we might have a problem.

Reply:
Ok. I would rather download the file and store on the library's github page; however, I do not know the policy on file storage for datatable.

Reply:
Also good idea, we can just put it under the `docs/_static`.